src/fastoad/notebooks/02_explanation/01_summary.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<p>"In order to do so,"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Doc links should be modified:</p><ul><li>for FAST-OAD, please simply link to https://fast-oad.readthedocs.io/</li><li>for OpenMDAO, I think a link to https://openmdao.org/newdocs/versions/latest/main.html would be more appropriate</li></ul>

Reply:
Done

Reply:
Corrected in commit 5f8b3871, also used as a tests on how to reply to all the comments